server/lib/packets/login.go

```diff
@@ -50,18 +49,18 @@ func ProcessLogin(from Source, msgid uint32, req *proto.LoginRequest, t Transpor
 		log.Printf("login database %v %v", a.User, a.Secret);
 		if _, err := a.Save(models.DBM(), []string{ "Secret", "Pass" }); err != nil {
 			log.Printf("login update account database error: %v", err)
-			SendError(from, msgid, proto.Error_Login_DatabaseError)
+			SendError(from, msgid, &proto.Err{ Type: proto.Error_Login_DatabaseError })
```

Comment:
__[gometalinter]__
* [gotype] undeclared name: proto
* [gotype] cannot pass argument &(proto.Err literal) (value of type *invalid type) to parameter of type error